cicecore/drivers/standalone/cice/CICE_RunMod.F90

```diff
@@ -627,11 +627,12 @@ subroutine sfcflux_to_ocn(nx_block,   ny_block,     &
       
       real (kind=dbl_kind)    :: &
           puny, &          !
+          Lsub, &          !
           rLsub            ! 1/Lsub
 
       character(len=*), parameter :: subname = '(sfcflux_to_ocn)'
 
-      call icepack_query_parameters(puny_out=puny)
+      call icepack_query_parameters(puny_out=puny, Lsub_out=Lsub)
```

Comment:
This is done in ice_forcing.F90. Why does it need to be here?

Reply:
If you mean what is the point of the whole subroutine, I'm not yet familiar with all of the intricacies of how coupling works, so I don't know the specifics. The comments at the top of the subroutine state the following:

~~~
! If surface heat fluxes are provided to CICE instead of CICE calculating
! them internally (i.e. .not. calc_Tsfc), then these heat fluxes can 
! be provided at points which do not have ice.  (This is could be due to
! the heat fluxes being calculated on a lower resolution grid or the
! heat fluxes not recalculated at every CICE timestep.)  At ice free points, 
! conserve energy and water by passing these fluxes to the ocean.
~~~


Here I'm just making sure the code compiles fine when `CICE_IN_NEM` is defined

Reply:
> No, I meant the Lsub initialization. This is done in ice_forcing.F90.

Ok. Here I'm ont initializing Lsub, just using it. The variable was not declared in that subroutine.It had not been catched because none of our tests compile with CICE_IN_NEMO.

Reply:
Got it. I'm not familiar with the CICE_IN_NEMO code pieces.

Reply:
@dabail10 This entire subsection only applies when calc_Tsfc is false and CICE_IN_NEMO is on